src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+
+  wall.style.position = 'relative';
+
+  const rect = wall.getBoundingClientRect();
+
+  let x = e.clientX - rect.left;
+  let y = e.clientY - rect.top;
+
+  if (x > 400) {
```

Comment:
what are magical numbers 400, 15, 35?

Create variables like `const wallX = rect.clientWidth;`
Same for spider size (and use spiderSize / 2, etc.)

Because now it's not clear why you used these exact numbers